laravel/app/Http/Controllers/ReportController.php

```diff
@@ -51,7 +53,7 @@ public function searchUsers(Request $request)
                 'id' => $user->id,
                 'name' => $user->name,
                 'full_name' => $user->data()->exists() && $user->data->full_name ? $user->data->full_name : '',
-                'burner_name' => $user->data()->exists() && $user->data->burner_name ? $user->data->burner_name : '',
+                'burner_name' => Helpers::displayName($user,''),
```

Comment:
It's redundant to add empty quotes to all of these function calls, since there is already a default value of "null". Why include them?

Reply:
The aim was mostly to keep it idempotent, but you're right this looks a little messy for no reason.

The only extra case I could think of is that there could have been some functionality after that made existance checks for burner_name that could fuddled it.

I'll go ahead and correct those!